src/NetEscapades.EnumGenerators/NetEscapades.EnumGenerators.csproj

```diff
@@ -22,5 +22,7 @@
     <None Include="$(OutputPath)\NetEscapades.EnumGenerators.Attributes.dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
     <None Include="$(OutputPath)\NetEscapades.EnumGenerators.Attributes.dll" Pack="true" PackagePath="lib\netstandard2.0" Visible="true" />
     <None Include="$(OutputPath)\NetEscapades.EnumGenerators.Attributes.xml" Pack="true" PackagePath="lib\netstandard2.0" Visible="true" />
+    <None Include="$(OutputPath)\NetEscapades.EnumGenerators.Attributes.dll" Pack="true" PackagePath="lib\net451" Visible="true" />
```

Comment:
You need `lib\net461`

Reply:
🤦‍♂️ I meant to make the target `net451` not `net461`, thanks

Reply:
Fixed in https://github.com/andrewlock/NetEscapades.EnumGenerators/commit/85f8226442a7ce5ce4b75e3e012ba60edaa80f8c